region/maridia/outer.json

```diff
@@ -1147,12 +1161,57 @@
                 ]}
               ],
               "unlock": null
+            },
+            {
+              "id": 9,
+              "requires": [
+                "canSuitlessMaridia",
+                {"canComeInCharged":{
+                  "fromNode": 3,
+                  "framesRemaining": 0,
+                  "shinesparkFrames": 70
+                }}
+              ],
+              "unlock": null,
+              "note": "One-way link for diagonal suitless shinesparking."
+            },
+            {
+              "id": 10,
+              "requires": [
+                {"or": [
+                  "canSwim",
+                  "canSuitlessMaridia"
+                ]},
+                {"canComeInCharged":{
+                  "fromNode": 3,
+                  "framesRemaining": 0,
+                  "shinesparkFrames": 65
+                }}
+              ],
+              "unlock": null,
+              "note": "One-way link for Shinesparking."
             }
```

Comment:
I'm going to hate myself for pointing this out, but there's also a way to spark directly up to 4 with Gravity and some walljumping. Seems to require about 120 frames. It's a diagonal spark right, from just a bit to the left of the right hill peak.

Reply:
....Actually no one cares cause if you can get there you can drop down to 2 and use that. Never mind!

Reply:
Phew dodged one!